index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Дизайн-студия "Нёрдс"</title>
+  </head>
+  <body>
+     <header class="main-header">
+        <nav class="main-navigation">
+            <a class="main-header-logo">
+                <img src="img/index-logo.svg" width="368" height="153" alt="Логотип дизайн-студии «Нёрдс»">
+            </a>
+            <ul class="site-navigation">
```

Comment:
```suggestion
            <ul class="navigation">
```
слово `site` не нужно указывать, это можно сказать тавтология, и так понятно что это сайт